ts/src/pro/ascendex.ts

```diff
@@ -313,8 +313,8 @@ export default class ascendex extends ascendexRest {
         //
         // ["40990.47","0.01619"],
         //
-        const price = this.safeFloat (delta, 0);
-        const amount = this.safeFloat (delta, 1);
+        const price = this.safeNumber (delta, 0);
```

Comment:
@samgermain I don't think this is enough,internally the orderbookside uses bisect left that is not expecting a string there
![image](https://user-images.githubusercontent.com/43336371/232789818-e37e84db-273a-486e-a288-84f393f6cad3.png)

@frosty00 can you confirm?

Reply:
To test this out, do I just need to set 

```
"options": {
            "number": "String"
 }
```

within my `keys.local.json`?

Reply:
@carlosmiei It works fine actually

This script

```ts
import ascendex from '/Users/samgermain/Documents/CCXT/ccxt/ts/src/pro/ascendex.js';
import keys from './keys.local.json' assert { type: "json" };

async function main () {

    const ascendexEx = new ascendex ({'apiKey': keys.ascendex.apiKey, 'secret': keys.ascendex.secret});
    // @ts-ignore
    ascendexEx.number = String; // @ts-ignore

    await ascendexEx.loadMarkets ();
    const ascendexBook = await ascendexEx.watchOrderBook ('BTC/USDT');
    console.log (ascendexBook);

};

main ();
```

prints

```
% node --loader ts-node/esm example.ts
OrderBook {
  bids: Bids(136) [
    [ '29539.99', '0.19602' ],
    [ '29537.22', '0.0068' ],
    ...
    [ '30455', '1' ],
    [ '30470', '1' ],
    ... 203 more items
  ],
  timestamp: 1691617816607,
  datetime: '2023-08-09T21:50:16.607Z',
  nonce: 8077974,
  symbol: 'BTC/USDT'
}
```